source/common_fields.rst

```diff
@@ -0,0 +1,484 @@
+- You can define whether this group can be **visible in a ticket** (as a requester, observer, assigned to, task, can be notified)
+- Define if the group can be a **manager in a project**
+- Select if this group **can contain** **Items** and/or **Users**
+- Click **+ Add**
+
+.. image:: ../image/add-group-1.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 63%
+
+.. image:: ../image/add-group-2.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 63%
+
+- In this window, you need to add users. In **Users** tab, select those you wish to associate with the group (prefer users with a technical or higher profile)
+
+.. image:: ../image/add-users-in-group-charge.png
+   :alt: module assets - add user in group in charge
+   :align: center
+   :scale: 43%
+
+1. Select **user**
+2. Define it this user can be **Manage** group (add/modify memebers, name, etc.)
+3. whether or not to authorise **Delegatee** rights (can open ticket for the group)
+4. **Add** your group
+
+Once the group has been created, you can add another or select it from the drop-down list.
+
+.. Note:: If you cant another option with groups, see :doc:`../administration/groups`
+
+Inventory number
+~~~~~~~~~~~~~~~~
+
+**Inventory number** is information added manually. It is generally managed internally by the company.
+
+Location
+~~~~~~~~
+
+.. image:: ../image/location.png
+   :alt: module assets - location
+   :align: center
+   :scale: 50%
+
+You can add location manually or dynamically.
+
+.. warning:: Depending on your needs, pay attention to the entity you are in and its recursivity, so that it is visible or not to other entities.
+
+.. Note:: All locations your created here are available for other GLPI objects depending on the entity in which you created it
+
+To add a location **manually** :
+
+If your have already entered locations :
+
+* Select the one that is right for you in the drop-down list.
+
+If you have not previously entered a location :
+
+* Click on **+**
+* In the new box, enter the informations that you need :
+
+  * Name
+  * Code
+  * As child of
+  * Postal code
+  * state
+  * Building number
+  * Longitude
+  * Latitude
+  * Altitude
+  * Comments
+  * Aliras
+  * Address
+  * Town
+  * Country
+  * Room number
+
+.. image:: ../image/add-location-1.png
+   :alt: module assets - add location
+   :align: center
+   :scale: 61 %
+.. image:: ../image/add-location-2.png
+   :alt: module assets - add location
+   :align: center
+   :scale: 61 %
+
+* Click **+ Add**
+* After saving, you can add a new location or close this window
+* Now, select the location for your computer
+
+
+To add a location **automatically** :
+
+You need to create **location rules** to affect a location to a computer. Before, you need to define the criteria which will trigger the rule (a tag inventory, a subnet, a domain, etc.)

```

Comment:
Maybe a better place would be with other rules in admin module (with a link to it here).